libs/checkout/src/drivers/interfaces/order-service.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Observable } from 'rxjs';
+
+import { Order } from '../../models/order/order';
+
+export interface DaffOrderServiceInterface {
```

Comment:
If we were to eventually have an "orders" module which handles the process of order management, would this cause a namespace conflict?

Reply:
no, because they would be imported from different places, `@daffodil/checkout` and `@daffodil/order`